addons/l10n_sa_edi/i18n/ar.po

```diff
@@ -920,13 +920,18 @@ msgstr "المحافظة / التقسيمات الدولية"
 
 #. module: l10n_sa_edi
 #: model_terms:ir.ui.view,arch_db:l10n_sa_edi.sa_partner_address_form
-msgid "State..."
-msgstr "المحافظة..."
+msgid "State"
+msgstr "المحافظة"
 
 #. module: l10n_sa_edi
 #: model_terms:ir.ui.view,arch_db:l10n_sa_edi.sa_partner_address_form
-msgid "Street"
-msgstr "الشارع"
+msgid "Street..."
+msgstr "الشارع..."
```

Comment:
This mindfucked me completely.
Pycharm shows it with the `...` on the left while github on the right.
So I thought you were doing it wrong as I was looking in Pycharm for the terms in `base` (which I suppose come from Arabish speaking people on Transifex).
But it's the same, so it's fine